sapl/templates/parlamentares/parlamentar_perfil_publico.html

```diff
@@ -30,38 +30,59 @@
       </div>
 
       <div class="col-sm-8">
-        <div id="div_data_nascimento" class="form-group">
+        <div id="div_partido" class="form-group">
           <p><b>Partido: </b> &nbsp {{object.filiacao_atual|default_if_none:"Não informado"}}</p>
         </div>
       </div>
 
       <div class="col-sm-8">
         <div id="div_data_nascimento" class="form-group">
-          <p><b>Data de Nascimento: </b> &nbsp {{object.data_nascimento|default_if_none:"Não informado"}}</p>
+          {% if not app_config.mostrar_campos_vazios_perfil_parlamentar and not object.data_nascimento %}
+            {{ pass }}
+          {% else %}
+            <p><b>Data de Nascimento: </b> &nbsp {{object.data_nascimento|default_if_none:"Não informado"}}</p>
+          {% endif %}           
         </div>
       </div>
 
       <div class="col-sm-8">
-        <div id="div_data_nascimento" class="form-group">
-          <p><b>Telefone: </b> &nbsp {{object.telefone|default_if_none:"Não informado"}}</p>
+        <div id="div_telefone" class="form-group">
+          {% if not app_config.mostrar_campos_vazios_perfil_parlamentar and not object.telefone %}
+            {{ pass }}
+          {% else %}
+            <p><b>Telefone: </b> &nbsp {{object.telefone|default:"Não informado"}}</p>
+          {% endif %}          
         </div>
       </div>
 
       <div class="col-sm-8">
-        <div id="div_data_nascimento" class="form-group">
-          <p><b>E-mail: </b> &nbsp {{object.email|default_if_none:"Não informado"}}</p>
+        <div id="div_email" class="form-group">
+          {% if not app_config.mostrar_campos_vazios_perfil_parlamentar and not object.email %}
+            {{ pass }}
+          {% else %}
+            <p><b>E-mail: </b> &nbsp {{object.email|default:"Não informado"}}</p>
+          {% endif %}
         </div>
       </div>
 
+      
       <div class="col-sm-8">
-        <div id="div_data_nascimento" class="form-group">
-          <p><b>Número do Gabinete: </b> &nbsp {{object.numero_gab_parlamentar|default_if_none:"Não informado"}}</p>
+        <div id="div_num_gabinete" class="form-group">
+          {% if not app_config.mostrar_campos_vazios_perfil_parlamentar and not object.numero_gab_parlamentar %}
+            {{ pass }}
+          {% else %}
+            <p><b>Número do Gabinete: </b> &nbsp {{object.numero_gab_parlamentar|default:"Não informado"}}</p>
+          {% endif %}
         </div>
       </div>
-
+      
       <div class="col-sm-8">
         <div id="div_biografia" class="form-group">
-          <p><b>Biografia: </b> &nbsp {{object.biografia|safe}}</p>
+          {% if not app_config.mostrar_campos_vazios_perfil_parlamentar and not object.biografia %}
+            {{ pass }}
+          {% else %}
+            <p><b>Biografia: </b> &nbsp {{object.biografia|safe}}</p>
+          {% endif %}
```

Comment:
```suggestion
          {% if "mostrar_campos_vazios_perfil_parlamentar"|get_config_attr or object.biografia %}
            <p><b>Biografia: </b> &nbsp {{object.biografia|safe}}</p>
          {% endif %}
```
Não se esqueça de adicionar la no inicio do arquivo o
```
{% load common_tags %}
```